server/controllers/request.js

```diff
@@ -0,0 +1,58 @@
+const User = require("../models/User");
+const Sitter = require("../models/Sitter");
+const Request = require("../models/Request");
+
+const asyncHandler = require("express-async-handler");
+
+// @route POST /request/creat
+exports.createRequest = asyncHandler(async (req, res, next) => {
+  const userId = req.user.id;
+
+  const { sitterId } = req.params.sitterId;
```

Comment:
You can send the `sitterId` in the request body as well. We only use path parameters when we want to identify a resource within our DB :)